app/views/users/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<% provide(:title, "My Profile") %>
+<h1>Welcome <%= "#{@user.name}" %>!</h1>
```

Comment:
If I try to access nonexisting user it should divert me, it shouldn't break the code.
Please fix this.